app/tests/__mocks__/clinics-with-ids-mock.json

```diff
@@ -0,0 +1,101 @@
+[
```

Comment:
Notes for reviewers: This mock file is used for tests that are included later in the series of stacked PRs. It mocks `/public/clinic-output/clinics-with-ids.json`